cypress/integration/test1.spec.js

```diff
@@ -0,0 +1,14 @@
+describe('ut2019', function() {
+    const languagesHome = ['https://ut2019.voog.com/en', 'https://ut2019.voog.com/et'];
```

Comment:
You could use the base url https://ut2019.voog.com as a configurable variable and not duplicate it across the tests. In a real-world scenario you most probably will run these tests against different environments and it will help you to easily re-configure your tests against dev/test environments.